images/microOS/config.xml

```diff
@@ -560,31 +117,21 @@
         <type
             image="iso"
             primary="true"
-            flags="dmsquash"
+            flags="overlay"
             hybridpersistent_filesystem="ext4"
             hybridpersistent="true"
-            kernelcmdline="quiet systemd.show_status=yes console=ttyS0,115200 console=tty0 net.ifnames=0 \$ignition_firstboot ignition.platform.id=qemu"
-            bootkernel="custom"
-            devicepersistency="by-uuid"
-        >
-            <systemdisk>
-                <volume name="home"/>
-                <volume name="root"/>
-                <volume name="tmp"/>
-                <volume name="opt"/>
-                <volume name="srv"/>
-                <volume name="boot/grub2/i386-pc"/>
-                <volume name="boot/grub2/x86_64-efi" mountpoint="boot/grub2/x86_64-efi"/>
-                <volume name="boot/writable"/>
-                <volume name="usr/local"/>
-                <volume name="var" copy_on_write="false"/>
-            </systemdisk>
-            <machine memory="512" guestOS="suse" HWversion="4">
-                <vmdisk id="0" controller="ide"/>
-                <vmnic driver="e1000" interface="0" mode="bridged"/>
-            </machine>
-        </type>
+            kernelcmdline="quiet systemd.show_status=yes console=ttyS0,115200 console=tty0"
+            devicepersistency="by-uuid"/>
     </preferences>
+    <users>
+         <user password="suse1234" home="/root" name="root" groups="root" pwdformat="plain"/>
```

Comment:
I'd prefer if we were to keep solely the `aquarium` user here with a default password, and add it to the sudoers list in `config.sh`, instead of adding a default `root` password. That way we'd have less stuff to ask the user to change on first run.

Reply:
ok I will remove the root later. I keep that there for debugging in the beginning